docs/blocks-games/cat-jumper.md

```diff
@@ -505,7 +505,6 @@ Hops_and_Paw = sprites.create(img`
     . . . . f . f . . . f . f . . .
     `, SpriteKind.Player)
 controller.moveSprite(Hops_and_Paw, 80, 0)
-Hops_and_Paw.setFlag(SpriteFlag.BounceOnWall, false)
```

Comment:
Remove bounce, not replace with setBounce?

Reply:
Yes, I didn't see it anyway else in the file so it was a no op I think / bounce on wall would be off by default anyways. I'll double check, i might have missed something!